fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -73,9 +85,26 @@ def __init__(self, fapd_manager):
     def analyze_button_sensitivity(self):
         return self.analysis_available
 
+    def on_next_system(self, system):
+        profilerDict = system["profiler"]
+        self.update_field_text(profilerDict)
+
+    def update_field_text(self, profilerDict):
+        prefixes = ["execute", "arg", "user", "dir", "env"]
+        if not profilerDict:
+            profilerDict = {prefix + "Text": "" for prefix in prefixes}
+        for prefix in prefixes:
+            buff = self.get_object(prefix + "Entry").get_buffer()
+            buff.set_text(profilerDict[prefix + "Text"],
+                          len(profilerDict[prefix + "Text"])
+                          )
```

Comment:
If you changed the widget names in the glade to mirror the keys of the `profilerDict` you'd be able to do something like

```python
for k,v in profilterDict:
    self.get_object(k).get_buffer().set_text(v, len(v))
```
```